tools/continuous-ingestor/README.md

```diff
@@ -26,36 +24,161 @@ A script to generate a continuous stream of records that are ingested into Times
     - If the `database-name` and `table-name` do not pre-exist in Timestream, the application will stop with an error message.
     - Use `--help` argument to see more options available in the Sample Continuous Data Ingestor Python Application.
      E.g. ```python3 timestream_sample_continuous_data_ingestor_application.py --help```
-    
-### Examples
-#### Single-threaded ingest
-Starts a single-threaded ingest process the continues until SIGINT signal (CTRL + C) is received.
+
+## Data Explained
+The tool generates for each instances 20 common host metrics and 5 process events. The host metrics are described in the following table:
+
+| Domain  | Metric | Value Range | Description | 
+|:---     |:---    |:-------:    |:---         | 
+| CPU | cpu_user | 0-100 | Random value, normal distributed around 85% for high cpu, 15% for low cpu, with a standard deviation of 10. With the additional parameters `--saw-signal-cpu` or `--sin-signal-cpu`, the load value is calculated as the appropriate signal value plus white noise with the defined signal to noise ratio. |
+| | cpu_system, cpu_iowait, cpu_steal, cpu_nice, cpu_si, cpu_hi | 0-1 | Uniform random value between 0 and 1, to represent a small cpu load < 1%|
+| | cpu_idle | 0-100 | Value is the remainder of ```100 - cpu_user - cpu_[other]``` to ensure that cpu load adds up to 100.|
+| Memory | memory_used | 0-8k | A uniform distributed value between 0 and 8000 representing the use of memory in e.g. MB. |
+| | memory_free | 0-8k | `8000 - memory_used` the remainder of available memory |
+| | memory_cached | 0-4k | A value between zero and 4000 representing the disk cache in e.g. MB |
+| Disk | disk_io_read | 0-25k | A uniform random value between 0-25k.|
+| | disk_io_write | 0-25k | A uniform random value between 0-25k.|
+| | disk_used | N +/-  | A host specific value moving normal distributed up/down by 5 GB.|
+| | disk_free | 15 +/-  | 15GB +/- the inverse of disk_used |
+| | file_descriptors_in_use | 2200 +/- | 2200 open files (mean) with a standard deviation of 500 |
+| Network | latency_per_read | 8 +/- | A normal distributed value of mean 8 and standard deviation 4 | 
+| | latency_per_write | 12 +/- | A normal distributed value of mean 12 and standard deviation 8 | 
+| | network_bytes_in | 0-1000 | A uniform distributed value between 0 and 1000 bytes | 
+| | network_bytes_out | 0-5000 | A uniform distributed value between 0 and 4000 bytes | 
+
+The generated process (event) metrics are described in the following table:
+| Domain  | Metric | Value Range | Description | 
```

Comment:
Need an empty line here to render the table

Reply:
added empty line before table. Cant verify, as it renders correct on github and vscode.